utils/appliance/__init__.py

```diff
@@ -628,39 +695,32 @@ def precompile_assets(self, log_callback=None):
         return status
 
     @logger_wrap("Backup database: {}")
-    def backup_database(self, log_callback=None):
+    def backup_database(self, database_path="/tmp/evm_db.backup", log_callback=None):
         """Backup VMDB database
 
         """
         log_callback('Backing up database')
-
-        with self.ssh_client as ssh:
-            status, output = ssh.run_rake_command(
-                'evm:db:backup:local -- --local-file /tmp/evm_db.backup --dbname vmdb_production')
-            if status != 0:
-                msg = 'Failed to backup database'
-                log_callback(msg)
-                raise ApplianceException(msg)
+        status, output = self.ssh_client.run_rake_command(
```

Comment:
You no longer need to expand the tuple, When you use the resulting object in an if statement, it acts like status != 0.